Patch-Source: https://sources.debian.org/src/isc-kea/1.5.0-2/debian/patches/0003-kea_admin_fix/ (modified)
--
From: Kea <isc-kea@packages.debian.org>
Date: Tue, 19 Feb 2019 12:39:35 +0000
Subject: kea_admin_fix

Removed the fallback to the build version of kea-admin since it will not exist on users hosts.

--- a/src/bin/admin/kea-admin.in
+++ b/src/bin/admin/kea-admin.in
@@ -50,8 +50,6 @@
 # Include the installed admin-utils.sh if available. Fallback to sources otherwise.
 if test -f "@datarootdir@/@PACKAGE_NAME@/scripts/admin-utils.sh"; then
     . "@datarootdir@/@PACKAGE_NAME@/scripts/admin-utils.sh"
-else
-    . "@abs_top_builddir@/src/bin/admin/admin-utils.sh"
 fi
 
 # Find the installed kea-lfc if available. Fallback to sources otherwise.
